schemas/manifest-schema.json

```diff
@@ -0,0 +1,152 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
```

Comment:
Use schema specification that is supported by Visual Studio: `http://json-schema.org/draft-07/schema`.

Reply:
Done.